components/feature/sitepermissions/src/main/res/layout/mozac_site_permissions_prompt.xml

```diff
@@ -0,0 +1,80 @@
+<RelativeLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        tools:ignore="Overdraw"
+        android:background="?android:windowBackground"
+        android:orientation="vertical">
+
+    <ImageView
+            android:id="@+id/icon"
+            android:layout_width="32dp"
+            android:layout_height="32dp"
+            android:layout_marginStart="16dp"
+            android:scaleType="center"
+            tools:src="@android:drawable/ic_menu_camera"
+            android:importantForAccessibility="no"
+            android:layout_alignParentTop="true"
+            android:layout_marginTop="16dp"
+            android:tint="?android:attr/textColorPrimary"
+            tools:tint="#000000"/>
+
+    <TextView
+            android:id="@+id/title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="11dp"
+            android:layout_marginStart="3dp"
+            tools:textColor="#000000"
+            android:textColor="?android:attr/textColorPrimary"
+            android:paddingTop="4dp"
+            android:paddingStart="5dp"
+            android:paddingEnd="5dp"
+            android:layout_alignParentTop="true"
+            android:layout_marginTop="16dp"
+            android:layout_toEndOf="@id/icon"
+            android:layout_alignBaseline="@id/icon"
+            tools:text="Allow wikipedia.org to use your camera?"/>
+
+    <CheckBox
+            android:id="@+id/do_not_ask_again"
```

Comment:
We might need text wrapping on this line or padding to make it fit in the dialog. CoordinatorLayout fixes this too! 🙃 

<img width="481" alt="Screen Shot 2019-05-06 at 4 00 07 PM" src="https://user-images.githubusercontent.com/1370580/57251606-39a2b000-7018-11e9-83ed-bee4b0cd8aca.png">

Reply:
I couldn't  understand what was the issue here. I was doing some tests and it's doing the text wrapping by default.

![Screenshot_20190506-220212](https://user-images.githubusercontent.com/773158/57266636-d2522380-704a-11e9-8674-712ae78953e7.png)

Reply:
I will merge this if there is any issue pending please feel free to open an issue and I will address it in a follow-up PR.